lib/node_modules/@stdlib/assert/is-float16array/lib/main.js

```diff
@@ -0,0 +1,57 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var nativeClass = require( '@stdlib/utils/native-class' );
+
+// VARIABLES //
+
+var hasFloat16Array = ( typeof Float16Array === 'function' ); // eslint-disable-line stdlib/require-globals
+
+
+// MAIN //
+
+/**
+* Tests if a value is a Float16Array.
+*
+* @param {*} value - value to test
+* @returns {boolean} boolean indicating whether value is a Float16Array
+*
+* @example
+* var Float16Array = require( '@stdlib/array/float16/lib/polyfill.js' );
```

Comment:
Generally we want to avoid loading internals of a package. Can we do `var Float16Array = require( '@stdlib/array/float16' )` here?